### ISSUE TITLE
`Tooltip` does not appear for overflowed text in `TextLine` - keyboard accessibility

### ISSUE THREAD

**Issue by spli02** [fortanix/baklava#378]
When navigating the page using a keyboard, the `Tooltip` does not appear for overflowed text in `TextLine` as the text is not focusable.

https://github.com/user-attachments/assets/1526ec55-bd2f-4530-8f15-afec2455c106

**Comment by mkrause** [comment]
Yes, this is intended. `TextLine` should be considered just text from an accessibility point of view, so not interactive. Screen readers will still read the entire text because the `text-overflow: ellipsis` is visual only. And visual users can just mouse over.

I think the one edge case here is touch input users (like mobile/tablet), who are visual but don't really have a way to do a mouse over. We could support something like a tap input to trigger the tooltip on touch devices. But not an accessibility concern in any case so I'll close this ticket.
